server/plugin.go

```diff
@@ -29,4 +33,24 @@ func (p *Plugin) ServeHTTP(c *plugin.Context, w http.ResponseWriter, r *http.Req
 	p.router.ServeHTTP(w, r)
 }
 
-// See https://developers.mattermost.com/extend/plugins/server/reference/
+func (p *Plugin) OnPluginClusterEvent(c *plugin.Context, ev model.PluginClusterEvent) {
+	if ev.Id != constants.ClusterEvent {
+		return
+	}
+
+	var event *serializer.UserStatus
+	if err := json.Unmarshal(ev.Data, &event); err != nil {
+		p.API.LogDebug("Error in unmarshaling the cluster event data", "Error", err.Error())
+		return
+	}
+
+	p.BroadcastEvent(event)
```

Comment:
what was our finding on whether to call it twice or not?

Reply:
Yes, we have to call it twice because the `PublishClusterEvent` function only publishes the event to other nodes except the current one.